liquibase-core/src/main/java/liquibase/change/core/RawSQLChange.java

```diff
@@ -10,29 +10,11 @@
 import liquibase.util.StringUtil;
 
 /**
- * Allows execution of arbitrary SQL.  This change can be used when existing changes are either don't exist,
+ * Allows execution of arbitrary SQL. This change can be used when existing changes are either don't exist,
  * are not flexible enough, or buggy. 
  */
-@DatabaseChange(name="sql",
-        description = "The 'sql' tag allows you to specify whatever sql you want. It is useful for complex changes " +
-            "that aren't supported through Liquibase's automated refactoring tags and to work around bugs and " +
-            "limitations " +
-            "of Liquibase. The SQL contained in the sql tag can be multi-line.\n" +
-            "\n" +
-            "The createProcedure refactoring is the best way to create stored procedures.\n" +
-            "\n" +
-            "The 'sql' tag can also support multiline statements in the same file. Statements can either be split " +
-            "using a ; at the end of the last line of the SQL or a 'GO' on its own on the line between the statements " +
-            "can be used. Multiline SQL statements are also supported and only a ; or GO statement will finish a " +
-            "statement, a new line is not enough. Files containing a single statement do not need to use a ; or GO.\n" +
-            "\n" +
-            "The sql change can also contain comments of either of the following formats:\n" +
-            "\n" +
-            "A multiline comment that starts with /* and ends with */.\n" +
-            "A single line comment starting with <space>--<space> and finishing at the end of the line.\n" +
-            "Note: By default it will attempt to split statements on a ';' or 'go' at the end of lines. Because of " +
-            "this, if you have a comment or some other non-statement ending ';' or 'go', don't have it at the end of a " +
-            "line or you will get invalid SQL.",
+@DatabaseChange(name = "sql",
+        description = "Allows you to directly specify whatever SQL you want.",
```

Comment:
I'd change this to, "Allows you to directly specify raw SQL to execute against the database."